homeassistant/helpers/script.py

```diff
@@ -295,6 +301,10 @@ async def async_validate_action_config(
     return config
 
 
+class _AbortScript(Exception):
+    """Throw if script needs to abort."""
```

Comment:
Can we clarify in the doc string of this + `StopScript` what is different (intention/cause)

Reply:
Fixes in [fa9d55b](https://github.com/home-assistant/core/pull/67340/commits/fa9d55ba0202f46c9752313c7c4c74d6eb4920e1)